### PR TITLE
use new license plate data from KBA

### LINKED ISSUE #5: switch to more up-to-date data source
Now after the "Kennzeichenliberalisierung" old prefixes are again useable.
That means that the current list of license plate prefixes is not up to date.

Can you update the list?
The problem is that I can't find a up to date list from the gouvernement.
This list seems to be relatively up to date.

### PR DESCRIPTION
**Pull request by b-vetter**
It makes sense now also to add the download and unzip functionality to get always the newest data.

edit by @derhuerst: closes #5

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+const source = './KFZ250.csv'
 const data = {}
 
-download.stream(source)
-.pipe(parser())
-.on('data', (entry) => {data[entry['Kennzeichen, Juli 2012']] = entry['Stadt bzw. Landkreis']})
+fs.createReadStream(source)
+.pipe(parser({"separator": ";"}))
+.on('data', (entry) => {
+	entry.KFZ.split("  ").forEach(function(char) {
```
Comment (derhuerst) [inline build.js]: Let's use `/\s+/` instead of `"  "` here.

**Review by derhuerst — COMMENTED** [review]
Thanks for your contribution!

Please keep the download from the upstream data source, and remove `KFZ250.csv` from the tree.

**Comment by b-vetter** [comment]
Done. ✅

**Comment by derhuerst** [comment]
Thanks!

In 409e00a, I've fixed a few problems in the build script:

- handle stream errors properly 🐛
- exit with 1 on errors 🐛
- check response status code & content-type header
- find `KFZ250.csv` file by file name, not by absolute path
- don't extract files that we don't need

**Pull request merged**